src/components/LetterForm.tsx

```diff
@@ -83,11 +85,6 @@ export function LetterFormDialogContent() {
             </button>
           </Dialog.Close>
         </div>
-        <Dialog.Close asChild>
```

Comment:
wait why

Reply:
you can just click off the modal and it closes it

Reply:
but accessibility